fhirpath_helpers/elasticsearch/mapping.py

```diff
@@ -50,16 +50,14 @@ def generate_mappings(fhir_release=None):
 
     elements_paths = build_elements_paths(resources_elements)
 
-    maps = dict()
-    if fhir_release == "STU3":
-        data_maps = fhir_data_types_maps_STU3
-    else:
-        data_maps = fhir_data_types_maps
-
+    mappings = dict()
+    fhir_es_mappings = fhir_types_mapping(
+        fhir_release, reference_analyzer, token_normalizer
+    )
     for resource, paths_def in elements_paths.items():
```

Comment:
Could you put the mappings "declaration" closer to the loop here?
Or maybe use a dict comprehension but I don't if it would be clearer

Reply:
done